execution_engine_testing/tests/src/test/system_contracts/upgrade.rs

```diff
@@ -117,23 +118,20 @@ fn should_allow_only_wasm_costs_patch_version() {
             .with_current_protocol_version(PROTOCOL_VERSION)
             .with_new_protocol_version(new_protocol_version)
             .with_activation_point(DEFAULT_ACTIVATION_POINT)
-            .with_new_wasm_config(new_wasm_config)
             .build()
     };
 
+    let engine_config = EngineConfig::new(5, new_wasm_config, SystemConfig::default());
```

Comment:
```suggestion
    let engine_config = EngineConfig::new(DEFAULT_MAX_QUERY_DEPTH, new_wasm_config, SystemConfig::default());
```

Reply:
Applied suggestion

Reply:
Looks like you didn't apply the suggestion.